source/components/molecules/CharacterCard/CharacterCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from "react";
+import { ThemeContext } from "styled-components/native";
+import Checkbox from "../../atoms/Checkbox";
+import { colorPalette } from "../../../styles/palette";
+
+import { Card, Section, Image, CardTitle, CardSubtitle } from "./styled";
+
+type CharacterCardType = {
+  icon: any;
```

Comment:
We should always avoid using `any` unless really necessary. Can `icon` be narrowed down?

Reply:
Found type for icon.Will use it.